rosa_getting_started/rosa-deleting-cluster.adoc

```diff
@@ -7,4 +7,13 @@ toc::[]
 
 Delete a {product-title} (ROSA) cluster using the `rosa` command-line.
 
+[id="prerequisites_rosa-deleting-cluster"]
+== Prerequisites
+
+* If {product-title} created a VPC, you must remove the following before you delete your cluster:
+  * network configurations, such as VPN configurations and VPC peering
+  * any additional services added to the VPC
```

Comment:
```suggestion
  * Any additional services that were added to the VPC[,such as $EXAMPLE_1 and $EXAMPLE_2]
```

Reply:
@0kashi what example services should I include?

Reply:
there are tons of aws services such as other ec2 instances, databases, network objects.  I don't know that we need to enumerate them.